### PR TITLE
feat: added modal and grid

### PR DESCRIPTION
**Pull request by sharath-io**
deploy preview:
https://deploy-preview-13--atom-ui.netlify.app/components/modal/modal.html
https://deploy-preview-13--atom-ui.netlify.app/components/grid/grid.html

files added:
modal.html
modal.css
modal.js

grid.html
grid.css

Screen Shot 2022-04-07 at 1 19 49 PM
Screen Shot 2022-04-07 at 1 19 54 PM

### CODE REVIEW

File: components/card/card.html [hunk 1]
```diff
@@ -87,8 +89,7 @@ <h2>Normal cards</h2>
...
-            style="width: 765px; height: 546px";
-            sandbox="allow-scripts allow-same-origin">
+            style="width: 765px; height: 546px" ; sandbox="allow-scripts allow-same-origin">
```
Comment (bharati-21) [inline components/card/card.html:92]: Please avoid inline stlyes/

File: components/card/card.html [hunk 2]
```diff
@@ -280,8 +276,7 @@ <h2>Horizontal card</h2>
...
-            style="width: 664px; height: 492px;"
-            sandbox="allow-scripts allow-same-origin">
+            style="width: 664px; height: 492px;" sandbox="allow-scripts allow-same-origin">
         </iframe>
 
     </div>
```
Comment (bharati-21) [inline components/card/card.html:282]: Please avoid inline styles.

Comment (sharath-io) [reply components/card/card.html:282]: This is from card component. Will update iframe styling later for P0 components

**Review by bharati-21 — APPROVED** [review]
Clean and well formatted code.

My only suggestion is that you could avoid inline styles.
Otherwise looks good.

**Review by reubenjnazareth — COMMENTED** [review]
Looks neat. Good to merge.

**Pull request merged**